src/variable.set_expiration.md

```diff
@@ -0,0 +1,50 @@
+## Set Expiration [/api/variables/{id}/expiration{?duration}]
+
+### Set an expiration timestamp for a variable [POST]
+
+:[min_version](partials/min_version_4.6.md)
+
+You can set an expiration time for variables in Conjur.
+After a variable expires, attempts to retrieve its value will fail.
+Variable expiration will generate an audit event.
+
+Variables may be marked with an annotation named `expiration/timestamp`.
```

Comment:
I think this should say something like:

> When a variable is set to expire, it will be marked with an annotation named `expiration/timestamp`. The value of this annotation will be an ISO8601 UTC timestamp